Sources/core/objects/Projectiles.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "Movable.h"
+#include "Human.h"
+
+
+class Projectile : public IMovable
+{
+public:
+    Projectile(size_t id);
+    DECLARE_SAVED(Projectile,IMovable)	
+    DECLARE_GET_TYPE_ITEM(Projectile)
+    void MakeMovementPattern(VDir target,id_ptr_on<Human> x);
+    void Process();
+    virtual void Bump(id_ptr_on<IMovable> item) override;
+    virtual void BumpByGas(Dir dir, bool inside = false) override {};
+    virtual bool CheckPassable() override;
+    virtual void AfterWorldCreation() override;
+    void MakeMovementLoops(int straight,int diagonal,VDir Direction,VDir Direction2);
+    Dir CalculateTrajectoryMove();
+    int KV_SAVEBLE(damage);
+private:
```

Comment:
It is better to make `damage` private and make public `GetDamage` function.
 I know, that this rule (and some others) does not always make place for my code, but that means that it is wrong code and this is should be fixed.